pkgs/tools/security/dkimpy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, pythonPackages, fetchurl, openssl, makeWrapper }:
```

Comment:
@Mic92, @FRidh: I think this is what you both intended? 
- code in development/python-modules/dkimpy
- callPackage in pythonPackages
- alias in all-packages